examples/lock-app/efr32/src/LockManager.cpp

```diff
@@ -77,6 +99,64 @@ CHIP_ERROR LockManager::Init(chip::app::DataModel::Nullable<chip::app::Clusters:
     return CHIP_NO_ERROR;
 }
 
+bool LockManager::IsValidUserIndex(uint16_t userIndex)
+{
+    if (userIndex > kMaxUsers)
```

Comment:
```cpp
return (userIndex < kMaxUsers);
```

I believe you are off-by-one here: if userIndex== kMaxUsers this is only valid for one-indexed however checking below I see that indices are already decremented by the time these are called.

This comment applies to several calls below as well.

Reply:
Great catch

Reply:
switched to a return (userIndex < kMaxUsers); notation